plugins/destination/duckdb/client/client.go

```diff
@@ -41,6 +44,18 @@ func New(ctx context.Context, logger zerolog.Logger, spec []byte, _ plugin.NewCl
 	if err != nil {
 		return nil, fmt.Errorf("failed to create batch writer: %w", err)
 	}
+
+	if strings.HasPrefix(c.spec.ConnectionString, "md:") {
+		// Motherduck, add 'custom_user_agent' to the connection string
+		if strings.Contains(c.spec.ConnectionString, "?") {
+			c.spec.ConnectionString += "&"
+		} else {
+			c.spec.ConnectionString += "?"
+		}
+		c.spec.ConnectionString += fmt.Sprintf("custom_user_agent=%s_%s_%s/%s(%s_%s)",
```

Comment:
Supposed to result in:
> custom_user_agent=cloudquery_destination_duckdb/1.2.3(darwin_ard64)

Ref: https://motherduck.com/docs/integrations/how-to-integrate